python/mxnet/gluon/block.py

```diff
@@ -926,6 +928,11 @@ def _get_graph(self, *args):
                 out = self.hybrid_forward(symbol, *grouped_inputs, **params)  # pylint: disable=no-value-for-parameter
             out, self._out_format = _flatten(out, "output")
 
+            if self._backend:
+                # To do: pass in all arguments
```

Comment:
Will this be addressed in this PR or later?

Reply:
Yes, it will be addressed in this PR. we're working on it now